common/clean_common.sh

```diff
@@ -385,3 +387,12 @@ function delete_cnis_bins_and_confs {
     rm -rf ${CNI_CONF_DIR}/*
     rm -rf ${CNI_BIN_DIR}/*
 }
+
+function clear_mlnx_vfs {
+    local mlnx_interfaces=$(ls -l /sys/class/net/ | grep -E $(lspci |grep Mellanox | grep -v Virtual | grep -Ev 'MT27500|MT27520' \
```

Comment:
I can't understand what happening here without playing with the command. 
Is it possible to split this command to separate steps?
As alternative option we can add comment which will show which part of lspci output we are trying to use.

Reply:
Done